gatsby-config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     author: `@gatsbyjs`,
   },
   plugins: [
+    'gatsby-plugin-sass',
```

Comment:
Should only be one `gatsby-plugin-sass` here, right? (Looks like a merge conflict that didn't get fixed.)

Might need to check your Prettier setup so that you're formatting quotes in a way that matches the master branch - that might have allowed Git to see that these are really two of the same line and avoided the merge conflict entirely.

Reply:
We removed the duplicate plugin. And got rid of the odd usage of backticks for apostrophes.